File: src/main/java/com/gary/Student/Student.java
package com.gary.Student;

public class Student {
    String name;
    String id;
    int math;
    int english;

    public Student(String name, int math, int english) {
        this.name = name;
        this.math = math;
        this.english = english;
    }

    public void print(){
        System.out.println(name + "\t" + math + "\t" + english
         + "\t" + (english + math )/2 );
    }
}

File: src/main/kotlin/com/gary/student/Student.kt
package com.gary.student

import java.util.*

fun main(args: Array<String>) {
    println("Please enter student's name : ")
    var scanner = Scanner( System.`in` )
    val name = scanner.next()

    println("Please enter student's English score : ")
    val english = scanner.nextInt()

    println("Please enter student's Math score : ")
    val math = scanner.nextInt()

    val stu = Students(name, math, english)
    stu.print()
}

class Students(var name : String , var math : Int, var english :Int){
    fun print(){
        println(name + "\t" + math + "\t" + english + "\t" + ( english + math)/2 )
    }
}